File: src/classes/api/req_res/get-kyous-request.ts
'use strict'

import { FindKyouQuery } from '../find_query/find-kyou-query'
import { GkillAPIRequest } from '../gkill-api-request'

export class GetKyousRequest extends GkillAPIRequest {

    query: FindKyouQuery

    constructor() {
        super()
        this.query = new FindKyouQuery()
    }

}

File: src/app/gkill/api/req_res/get_kyous_request.go
package req_res

import "github.com/mt3hr/gkill/src/app/gkill/api/find"

type GetKyousRequest struct {
	SessionID string `json:"session_id"`

	Query *find.FindQuery `json:"query"`

	LocaleName string `json:"locale_name"`
}
